frontend/src/models/models.html

```diff
@@ -47,18 +47,18 @@
             Fields
           </button>
           <span class="isolate inline-flex rounded-md shadow-sm">
-            <button type="button" class="relative inline-flex items-center rounded-none rounded-l-md bg-white px-2 py-2 text-gray-400 ring-1 ring-inset ring-gray-300 hover:bg-gray-50 focus:z-10">
+            <button @click="outputType = 'table'" type="button" class="relative inline-flex items-center rounded-none rounded-l-md bg-white px-2 py-2 text-gray-400 ring-1 ring-inset ring-gray-300 hover:bg-gray-50 focus:z-10">
               <img class="h-5 w-5" src="images/table.svg">
             </button>
-            <button type="button" class="relative -ml-px inline-flex items-center rounded-none rounded-r-md bg-white px-2 py-2 text-gray-400 ring-1 ring-inset ring-gray-300 hover:bg-gray-50 focus:z-10">
+            <button @click="outputType = 'json'" type="button" class="relative -ml-px inline-flex items-center rounded-none rounded-r-md bg-white px-2 py-2 text-gray-400 ring-1 ring-inset ring-gray-300 hover:bg-gray-50 focus:z-10">
               <img class="h-5 w-5" src="images/json.svg">
             </button>
           </span>
         </div>
       </div>
     </div>
     <div class="documents-container">
-      <table>
+      <table v-if="outputType == 'table' && status != 'loading'">
```

Comment:
This seriously screws up infinite scroll. If you scroll down to near the bottom of the list of documents (either table or json, doesn't matter), all the documents disappear and you lose your place.

Do we even need the `status !== 'loading'` check here? Can probably get rid of that. I just want the `loader.gif` image to show up at the bottom of the list if there's more documents loading